src/conditions/datadog_search.rs

```diff
@@ -305,7 +305,7 @@ impl Filter<LogEvent> for EventFilter {
 
 /// Returns a `Matcher` that returns true if the log event resolves to a string or
 /// numeric which matches the provided `func`.
```

Comment:
```suggestion
/// Returns a `Matcher` that returns true if the field resolves to a string,
/// numeric, or boolean which matches the provided `func`.
```
(also the function below could use the same "log event" => "field")